src/approximations/cvi.jl

```diff
@@ -25,26 +25,37 @@ Arguments
  - `n_samples`: number of samples to use for statistics approximation
  - `num_iterations`: number of iteration for the natural parameters gradient optimization
  - `opt`: optimizer, which will be used to perform the natural parameters gradient optimization step
+ - `grad`: optional, structure to select how the gradient and the hessian will be computed
```

Comment:
Mention, that `ForwardDiff` is used by default

Reply:
[Done](https://github.com/biaslab/ReactiveMP.jl/blob/cvi-stability-fix/src/approximations/cvi.jl#L28)